frontend/src/components/place/PlaceSidebarInfo.vue

```diff
@@ -79,20 +103,49 @@
 </template>
 
 <script>
+import moment from 'moment';
 import PlaceMapMarker from './PlaceMapMarker';
 
 export default {
     name: 'PlaceSidebarInfo',
-
     components: {
         PlaceMapMarker
     },
-
     props: {
         place: {
             type: Object,
             required: true
         }
+    },
+    data() {
+        return {
+            isShowSchedule: false
+        };
+    },
+    methods: {
+        isOpen() {
+            let today = moment().format('dddd');
+            let now = this.getMinutes(moment());
+            let isOpen = false;
+            _.forEach(this.place.worktime, (item) => {
```

Comment:
use a method `find` in order to find whether a place contains a today item and then check if it opened. It is better because you don't need to change a variable out of scope in such case

Reply:
Done.